flow-nodes/api-builder-plugin-md5/src/actions.js

```diff
@@ -1,19 +1,35 @@
 const crypto = require('crypto');
-
-function digest(req, outputs) {
-	const { data } = req.params;
+/**
```

Comment:
you should leave 1 empty line after requires and the rest of the code

Reply:
Fixed.